### PR TITLE
Migrate tests and lint suite to Nox

### PR DESCRIPTION
**Pull request by oncleben31**
Contribute to implement best practices of hypermodern_python #21 by using Nox.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -8,77 +8,102 @@ Vous trouverez les instructions en français [ici](#contribuer).
...
+Vous pouvez lancer une session Nox spécifique. Par exemple, lancez la suite de
+tests unitaires avec:
+
+`$ nox --session=tests`
+
+Les tests unitaires sont dans le répertoire `tests` et utilisent le framework pytest.
+
+### Soumettre votre Pull Request (PR)
+
+Oouvrez une [pull requet](https://github.com/hacf-fr/meteofrance-api/pulls) pour proposer des changements à ce projet.
```
Comment (Quentame) [inline CONTRIBUTING.md:48]: Ouvrez

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -8,77 +8,102 @@ Vous trouverez les instructions en français [ici](#contribuer).
...
--   You contribution is merged in the project. Thank you.
+Unit tests are located in the tests directory, and are written using the pytest testing framework.
 
-### Guidelines
+### How to submit changes
 
-To make maintenance and review easier, we recommand you to use some tools ensuring
-good code and format quality. With the librairies installed by
-`pip install -e .[testing]` command, you can use the following commands:
+Open a [pull requet](https://github.com/hacf-fr/meteofrance-api/pulls) to submit changes to this project.
```
Comment (Quentame) [inline CONTRIBUTING.md:99]: request

File: CONTRIBUTING.md (hunk 1 above)
Comment (oncleben31) [reply CONTRIBUTING.md:48]: Thanks will be take into account in next PR

File: CONTRIBUTING.md (hunk 2 above)
Comment (oncleben31) [reply CONTRIBUTING.md:99]: Will be take into account in next PR

**Pull request merged**